### DISCUSSION TITLE
(已解决)不是很懂为什么不能让游客请求刷新

### DISCUSSION THREAD

**Discussion by cancundeyingzi** [AlistGo/alist#1177] [Q&A] [answered]
难道有安全问题?还是怕频繁刷新炸服务器?还是有别的无法解决的问题。。。。

**Comment by anwen-anyi** [comment]
你让游客刷新你的API肯定会太频繁,然后不就会导致你的账号异常...轻则限速/限流 重则封号~

**Reply by cancundeyingzi** [reply]
> 你让游客刷新你的API肯定会太频繁,然后不就会导致你的账号异常...轻则限速/限流 重则封号~

这还不简单，设置最短刷新间隔，检测到5分钟内已经刷新过了，拒绝执行刷新命令。

**Reply by anwen-anyi** [reply]
> > 你让游客刷新你的API肯定会太频繁,然后不就会导致你的账号异常...轻则限速/限流 重则封号~
> 
> 这还不简单，设置最短刷新间隔，检测到5分钟内已经刷新过了，拒绝执行刷新命令。

呃呃呃...哈哈 我不会,大佬可以fork一份改改提一个pr贡献一下 哈哈~ QAQ

**Reply by cancundeyingzi** [reply]
> > > 你让游客刷新你的API肯定会太频繁,然后不就会导致你的账号异常...轻则限速/限流 重则封号~
> > 
> > 
> > 这还不简单，设置最短刷新间隔，检测到5分钟内已经刷新过了，拒绝执行刷新命令。
> 
> 呃呃呃...哈哈 我不会,大佬可以fork一份改改提一个pr贡献一下 哈哈~ QAQ

我issues提交新功能试过了，但是他直接给我关了，也没给理由，我不懂为什么关我的issues，才来这里提问

**Reply by cancundeyingzi** [reply]
> > > 你让游客刷新你的API肯定会太频繁,然后不就会导致你的账号异常...轻则限速/限流 重则封号~
> > 
> > 
> > 这还不简单，设置最短刷新间隔，检测到5分钟内已经刷新过了，拒绝执行刷新命令。
> 
> 呃呃呃...哈哈 我不会,大佬可以fork一份改改提一个pr贡献一下 哈哈~ QAQ

你要我改我也不会啊，我连python都是一行一谷歌，别说这个了

**Reply by anwen-anyi** [reply]
> > > > 你让游客刷新你的API肯定会太频繁,然后不就会导致你的账号异常...轻则限速/限流 重则封号~
> > > 
> > > 
> > > 这还不简单，设置最短刷新间隔，检测到5分钟内已经刷新过了，拒绝执行刷新命令。
> > 
> > 
> > 呃呃呃...哈哈 我不会,大佬可以fork一份改改提一个pr贡献一下 哈哈~ QAQ
> 
> 我issues提交新功能试过了，但是他直接给我关了，也没给理由，我不懂为什么关我的issues，才来这里提问

可能是重复了吧~ 应该有人提过类似的 `issue` 了

**Reply by cancundeyingzi** [reply]
> > > > > 你让游客刷新你的API肯定会太频繁,然后不就会导致你的账号异常...轻则限速/限流 重则封号~
> > > > 
> > > > 
> > > > 这还不简单，设置最短刷新间隔，检测到5分钟内已经刷新过了，拒绝执行刷新命令。
> > > 
> > > 
> > > 呃呃呃...哈哈 我不会,大佬可以fork一份改改提一个pr贡献一下 哈哈~ QAQ
> > 
> > 
> > 我issues提交新功能试过了，但是他直接给我关了，也没给理由，我不懂为什么关我的issues，才来这里提问
> 
> 可能是重复了吧~ 应该有人提过类似的 `issue` 了

我写的时候查过了，短时间内好像没人提这个

**Comment by xhofe** [comment, marked as answer]
这可能是个可选项 但不会是默认

觉得缓存时间长可以自己改。

**Reply by cancundeyingzi** [reply]
> 这可能是个可选项 但不会是默认
> 
> 觉得缓存时间长可以自己改。

所以为什么要定死呢？灵活不是更好吗？有人访问的时候自动刷新一下。每次刷新间隔最少五分钟或者几分钟。

**Reply by xhofe** [reply]
你将缓存时间改成5分钟就是这个效果。

不是会一直刷新 有人访问才会重新请求。

**Reply by cancundeyingzi** [reply]
> 你将缓存时间改成5分钟就是这个效果。
> 
> 不是会一直刷新 有人访问才会重新请求。

是么?我这网站居然有人在访问...我还以没人访问也会刷新,

搞错了
